ogcapi-features/extensions/transaction/README.md

```diff
@@ -24,14 +24,63 @@ OGC API [Simple Transactions](http://docs.opengeospatial.org/DRAFTS/20-002.html)
 once it is released STAC will align to a released one, but we anticipate few changes as it is a very simple document.
 
 STAC Transactions additionally support optimistic locking through use of the ETag header, as specified in the
-OpenAPI document. This is not currently specified in OGC API - Features, but it is compatible and we will 
+OpenAPI document. This is not currently specified in `OGC API - Features`, but it is compatible and we will 
```

Comment:
Code tag seems weird for a name?

Reply:
We've been doing it in other places too. The main problem is that it's unclear in a sentence that "OGC API - Features" is a proper name, but most people read it as parenthetical phrase. Maybe we should replace all of the instances of the names of conformance classes to be double-quoted rather than code quoted?

Reply:
Yeah, this is completely inconsistent throughout the spec. My first intuition though would be italic instead of code, which seems semantically better and probably also is better for accessibility.

Then it's

> This is not currently specified in *OGC API - Features*, but it is compatible and we will 

instead of

> This is not currently specified in `OGC API - Features`, but it is compatible and we will

Reply:
I'm going to leave this as it is and then we can do all that search and replace in one PR.